lib/account.php

```diff
@@ -83,6 +87,7 @@ public function __construct(MailAccount $account) {
 		$this->crypto = \OC::$server->getCrypto();
 		$this->config = \OC::$server->getConfig();
 		$this->memcacheFactory = \OC::$server->getMemcacheFactory();
+		$this->addressCollector = \OC::$server->query('OCA\Mail\Service\AutoCompletion\AddressCollector');
```

Comment:
@ChristophWurst this is the reason why this is falling apart.

In previous versions of owncloud you cannot query the server container for services which are located in an app ... let me fix that ...